help/develop/how-to/test-packages.md

```diff
@@ -97,55 +97,56 @@ This package contains a collection of test cases that can serve as illustrative
+* **Optional Flags**: You can use optional flags to customize the testing process:
+  * `--skip-build` skips the build step.
+  * `--skip-publish` skips the publishing step.
+  * `--csv` if you want to save test results. It saves the report as a CSV file in your package folder.
+  * `--gui` to see test execution in action. It disables the headless browser mode, aiding in test debugging.
+  
+  You can find all avalilable flags in this [table](https://github.com/datagrok-ai/public/blob/master/tools/README.md?plain=1#L122).
+  
 If you do not have any datagrok instance run locally, you can use [docker-compose](../../deploy/docker-compose.mdx) to run the stand.
 
 ### Trigger GitHub Actions manually
 
 If an error occurred for the action triggered by the commit, it is possible to trigger the action manually.
 
 1. Use [Packages workflow](https://github.com/datagrok-ai/public/actions/workflows/packages.yml)
-2. Press `run workflow` and set packages list to test separated with spaces, for example: `Demo Tutorials`. Choose the
-   target branch. Then `Run workflow`. Note that publish to the NPM registry is executed for the master branch only.
-3. Check that the GitHub Actions workflow finished successfully
-4. The results are available in the actions output
+2. Run the Workflow:
+    * Click on `Run workflow`.
+    * Set the list of packages you want to test, separating them with spaces (e.g., `Demo Tutorials`).
+    * Choose the target branch you want to test against.
+    * Finally, hit `Run workflow` button.
+    
+    Note that publish to the NPM registry is executed for the master branch only.
+3. Check that the GitHub Actions workflow finished successfully.
+4. The results are available in the actions output.
 
 ### Troubleshooting failed tests in GitHub Actions
 
-When a GitHub Action completes, you can access the "Artifacts" section on its main page ("Summary"), which contains a zip archive with the results of the tests for that specific action. This archive typically includes three important files that can help you diagnose and resolve the problems:
+When a GitHub Action completes, you can access the `Artifacts` section on its main page (Summary), which contains a zip archive with the results of the tests for that specific action. This archive includes three important files that can help you diagnose and resolve the problems:
 
-1. `test-console-output.log`: This file contains the console log records generated during   the test execution. The console output often includes valuable information such as error messages, stack traces, and debugging statements. It's a good starting point for understanding what went wrong during the test run.
+1. `test-console-output.log` contains the console log records generated during the test execution, including error messages, stack traces, and debugging statements. It's a good starting point for understanding what went wrong during the test run.
```

Comment:
I'll make it more simple and plain. Like:

> `test-console-output.log` contains the console output of the test execution.

The same applies to the next descriptions of files.

Reply:
What actionable/informational value does this sentence have?: "it's a good starting point...". Do we really need it?